src/ServerlessAutoSwagger.ts

```diff
@@ -151,9 +151,9 @@ class ServerlessAutoSwagger {
 
   predeploy = async () => {
     const stage = this.serverless.service.provider.stage;
-    const specifiedStages = this.serverless.service.custom?.autoswagger?.stages;
-    if(specifiedStages && specifiedStages.indexOf(stage) === -1) {
-      console.log(`Not deploying for stage: ${stage} due to custom.autoswagger.stages configuration.`);
+    const excludedStages = this.serverless.service.custom?.autoswagger?.excludeStages;
+    if(excludedStages && excludedStages.indexOf(stage) !== -1) {
+      console.log(`Not deploying for stage: ${stage} due to custom.autoswagger.excludeStages configuration.`);

```

Comment:
```suggestion
      console.log(`Swagger lambdas will not be deployed for stage [${stage}], as they have been marked for exclusion.`);
```

Reply:
^ The reason I want to change the wording is that it's important we specify that the Swagger lambdas will not be deployed. If `Not deploying for stage: ${stage}` prints out as part of the serverless deployment, it might confuse the user